tests/test_EclipseWriter.cpp

```diff
@@ -285,21 +285,21 @@ BOOST_AUTO_TEST_CASE(EclipseWriterIntegration) {
         std::vector<double> tranx(3*3*3);
         std::vector<double> trany(3*3*3);
         std::vector<double> tranz(3*3*3);
-        std::vector<data::CellData> eGridProps{
-            {"TRANX" , UnitSystem::measure::transmissibility, tranx},
-            {"TRANY" , UnitSystem::measure::transmissibility, trany},
-            {"TRANZ" , UnitSystem::measure::transmissibility, tranz}
-        };
+        CellDataContainer eGridProps({
+                    {"TRANX" , UnitSystem::measure::transmissibility, tranx},
+                    {"TRANY" , UnitSystem::measure::transmissibility, trany},
+                    {"TRANZ" , UnitSystem::measure::transmissibility, tranz}
+            });
 
         eclWriter.writeInitAndEgrid( );
         eclWriter.writeInitAndEgrid( eGridProps );
 
         data::Wells wells;
 
         for( int i = first; i < last; ++i ) {
-            std::vector<data::CellData> timesStepProps {
-                {"KRO" , UnitSystem::measure::identity , std::vector<double>(3*3*3 , i)},
-                {"KRG" , UnitSystem::measure::identity , std::vector<double>(3*3*3 , i*10)}};
+            CellDataContainer timesStepProps({
```

Comment:
This change is only mandated because `CellDataContainer` doesn't expose `vector`'s rich set of constructors any more, but I suppose that's fine.

Still though, I'd expect a type like this to support init lists if I'd touch upon it in templated code for example.